src/events/events.service.ts

```diff
@@ -12,8 +13,17 @@ export class EventsService {
     private readonly eventsRepository: Repository<Event>,
   ) {}
 
-  create(createEventInput: CreateEventInput) {
-    return this.eventsRepository.save(createEventInput);
+  create(createEventInput: CreateEventInput, user: User) {
+    const createEvent = new Event();
```

Comment:
Zamiast pojedynczego ustawiania atrybutów obiektu, można użyć destrukturyzacji (tak jak to zrobiłem w kolejnym commicie). Tworzenie pustego eventu też chyba nie ma sensu, bo ten przekazany  CreateEventInput już jest typu Event.